ca/certificate-authority.go

```diff
@@ -135,6 +134,7 @@ type CertificateAuthorityImpl struct {
 	SA               certificateStorage
 	PA               core.PolicyAuthority
 	Publisher        core.Publisher
+	GRPCTimeout      time.Duration
```

Comment:
Oops nevermind, I see that it is. :-( I'm not sure adding timeout fields to every service is the best way to handle this. Can timeout be an argument when constructing client wrappers?